package/src/components/Message/Message.tsx

```diff
@@ -648,6 +655,15 @@ const MessageWithContext = <
     setEditingState(message);
   };
 
+  const handlePinMessageUtil = async () => {
```

Comment:
The naming confuses me a little bit here, might something like `toggleMessagePinned` or similar be a bit clearer on what the function intends to do? I also think, in my personal opinion, that I'd rather pass in the message here like it's done with `handlePinMessage` farther down, but as there's no functional difference that's nothing to worry about imo

Reply:
As the prop name and function name was the same so I added `Util` in the end for this function after thinking much about it. Is it fine @vishalnarkhede?

Reply:
Lets keep it consistent with other function names that we have for similar case, such as `handleToggleMuteUser`, `handleToggleBanUser` etc., So in this case `handleTogglePinMessage`